using-mobile-extensions/places/places-usage-reference.md

```diff
@@ -0,0 +1,336 @@
+# AEPPlaces API Reference
+
+This document contains usage information for the public functions, classes, and enums in AEPPlaces.
+
+## Static functions
+
+- [clear](#clear)
+- [extensionVersion](#extensionVersion)
+- [getCurrentPointsOfInterest](#getCurrentPointsOfInterest)
+- [getLastKnownLocation](#getLastKnownLocation)
+- [getNearbyPointsOfInterest](#getNearbyPointsOfInterest)
+- [processRegionEvent](#processRegionEvent)
+- [registerExtension](#registerExtension)
+- [setAuthorizationStatus](#setAuthorizationStatus)
+
+---
+
+### clear
+
+Clears out the client-side data for Places in shared state, local storage, and in-memory.
+
+{% tabs %}
+{% tab title="Swift" %}
+**Signature:**
+```
+static func clear()
+```
+**Example Usage:**
```

Comment:
Sentence case, rather than title case. Also, we can get rid of the colon at the end, so "**Example usage**".

Reply:
should we also get rid of the colon for "**Signature:**?

Reply:
Yeah, I would do that as well.